components/controls/EscKeydownHandler.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Sonar UI Common
+ * Copyright (C) 2019-2019 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import { KeyCodes } from '../../helpers/keycodes';
+
+interface Props {
+  children: React.ReactNode;
+  onKeydown: () => void;
+}
+
+export default class EscKeydownHandler extends React.Component<Props> {
+  componentDidMount() {
+    setTimeout(() => {
```

Comment:
Why do we need this timeout at 0 ?

Reply:
Not sure, it was the case in `Toggler`, where this code basically comes from, so I thought I'd keep it, in case it was for some edge case somewhere.

Reply:
Ok that's weird we don't do such stuff in the other places where we bind key events... And I guess we lost the history about this when moving to sonar-ui-common...

Reply:
I believe this is a common trick to run code on another thread. But I don't understand why it is used here.

Reply:
Yeah... And this has an impact on many places (because of `Toggler`). So, I'm not confident  changing this is safe 😅